### PR TITLE
Add set_title to FileDialog

### PR DESCRIPTION
**Pull request by hatal175**
I've added a set_title function for windows, linux and mac. Doesn't seem like web-sys supports it (https://stackoverflow.com/questions/19944430/how-to-change-file-upload-dialogs-title-in-a-browser).

Important: I did not test for mac and linux, sorry for the incovenience.

### CODE REVIEW

**Comment by PolyMeilex** [comment]
Thank you! 
Looks good to me.
Works on Linux and Windows without any problems. I'll check macOS in my free time tomorrow.

**Comment by PolyMeilex** [comment]
I don't see a title on Mac, short google search tells me that those probably got removed in Mac OS X 10.11.
I think we can merge this as is, any toughs on that before we do?

**Comment by hatal175** [comment]
Maybe add something in the description about mac compatibility?

**Comment by PolyMeilex** [comment]
Yeah, you can add something like `(Only below version 10.11)` in `Supported Platforms` section of doc

File: src/dialog.rs [hunk 1]
```diff
@@ -73,6 +74,16 @@ impl FileDialog {
         self
     }
 
+    /// Set the title of the dialog.
+    /// #### Supported Platforms:
+    /// - Windows
+    /// - Linux
+    /// - Mac
```
Comment (PolyMeilex) [inline src/dialog.rs:81]:
```suggestion
    /// - Mac (Only below version 10.11)
```

File: src/dialog.rs [hunk 2]
```diff
@@ -172,6 +183,16 @@ impl AsyncFileDialog {
...
```
Comment (PolyMeilex) [inline src/dialog.rs:190]:
```suggestion
    /// - Mac (Only below version 10.11)
```

**Comment by PolyMeilex** [comment]
Thank you!

**Comment by PolyMeilex** [comment]
Published in `0.5.0`

**Pull request merged**